transformer/src/test/scala/uk/ac/wellcome/transformer/utils/RecordUtils.scala

```diff
@@ -0,0 +1,19 @@
+package uk.ac.wellcome.transformer.utils
+
+import com.amazonaws.services.dynamodbv2.model.{
+  AttributeValue,
+  Record,
+  StreamRecord
+}
+
+trait RecordUtils {
+
+  def createInvalidRecord: Record = {
+    val record = new Record()
+    val streamRecord = new StreamRecord()
+    streamRecord.addNewImageEntry("something",
+                                  new AttributeValue("something-else"))
```

Comment:
This method is to create an invalid record so they don't actually represent anything